myhpom/templates/myhpom/accounts/next_steps_no_ad_template.html

```diff
@@ -43,16 +43,16 @@ <h3>Welcome!</h3>
                         </p>
                         <form>
                             <div class="form-group">
-                                <label for="id_healthcare_network">Please share your healthcare network:</label>
+                                <label for="id_health_network">Please share your healthcare network:</label>
                                 <input
                                     class="form-control"
-                                    aria-describedby="healthcare_network_help"
+                                    aria-describedby="health_network_help"
                                     type="text"
-                                    name="healthcare_network"
-                                    id="id_healthcare_network"
+                                    name="health_network"
+                                    id="id_health_network"
```

Comment:
The manually entered network on this template isn't saved either - I think that should be addressed in this PR.